schema/address/validate.go

```diff
@@ -24,9 +24,9 @@ const lbrycrdTestnet = "lbrycrd_testnet"
 const lbrycrdRegtest = "lbrycrd_regtest"
 
 var addressPrefixes = map[string][2]byte{
-	lbrycrdMain:    [2]byte{lbrycrdMainPubkeyPrefix, lbrycrdMainScriptPrefix},
-	lbrycrdTestnet: [2]byte{lbrycrdTestnetPubkeyPrefix, lbrycrdTestnetScriptPrefix},
-	lbrycrdRegtest: [2]byte{lbrycrdRegtestPubkeyPrefix, lbrycrdRegtestScriptPrefix},
+	lbrycrdMain:    {lbrycrdMainPubkeyPrefix, lbrycrdMainScriptPrefix},
```

Comment:
ouch did you test this out? I'm always very scared of touching those things

Reply:
This syntax is fine (and it's also an auto gofmt fix)

Reply:
[fear is the mind-killer](https://dune.fandom.com/wiki/Litany_Against_Fear)